src/common/components/clusterWizard/ClusterWizardStepValidationsAlert.tsx

```diff
@@ -36,22 +38,37 @@ const ClusterWizardStepValidationsAlert = <ClusterWizardStepsType extends string
   wizardStepsValidationsMap,
   children,
 }: ClusterWizardStepValidationsAlertProps<ClusterWizardStepsType>) => {
-  const { failedClusterValidations } = React.useMemo(() => {
+  const { failedClusterValidations, failedHostValidations } = React.useMemo(() => {
     const reducedValidationsInfo = getWizardStepClusterValidationsInfo(
       validationsInfo || {},
       currentStepId,
       wizardStepsValidationsMap,
     );
+
+    const reducedHostValidationsInfo = hosts.map((host) => {
```

Comment:
I decided to use `checkHostValidation` as it is (with the boolean return type) because I'm only interested in the validation statuses (and whether they all pass or not) and not the messages.